cli/kubectl_kadalu/__main__.py

```diff
@@ -46,26 +54,12 @@ def main():
     """Handle Commands"""
     try:
         args = get_args()
-        if args.mode == "install":
-            install.validate(args)
-            install.run(args)
-        elif args.mode == "storage-add":
-            storage_add.validate(args)
-            storage_add.run(args)
-        elif args.mode == "storage-list":
-            storage_list.validate(args)
-            storage_list.run(args)
-        elif args.mode == 'storage-remove':
-            storage_remove.validate(args)
-            storage_remove.run(args)
+        func = cmds.get(args.mode)
```

Comment:
This can be simplified further. Remove the imports and dict definition.

```python
if args.mode == "version":
    return show_version()

try:
    mod = __import__(args.mode.replace("-", "_"))
    mod.validate(args)
    mod.run(args)
except ModuleNotFoundError:
    print('Invalid sub-command "%s"' % args.mode, file=sys.stderr)
    sys.exit(1)
```

Reply:
cool, will change the code.